airflow/plugins/operators/elavon_to_gcs_raw.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import gcsfs
+import paramiko
+
+from airflow.models import BaseOperator
+
+CALITP__ELAVON_SFTP_PASSWORD = os.environ["CALITP__ELAVON_SFTP_PASSWORD"]
+BIGQUERY_KEYFILE_LOCATION = os.environ["BIGQUERY_KEYFILE_LOCATION"]
+
+
+def mirror_raw_files_from_elavon():
+    """
+    Download Elavon transaction records from SFTP and write raw files to GCS for
+    further processing in another job
+    """
+
+    # Establish connection to SFTP server
+    client = paramiko.SSHClient()
+    client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    client.connect(
+        hostname="34.145.56.125",
+        port=2200,
+        username="elavon",
```

Comment:
can these be env vars so we don't have to deploy code to make changes?

although this is what [connections](https://cloud.google.com/composer/docs/how-to/managing/connections#creating_new_airflow_connections) are intended for; that might be preferable since any env vars would be managed in the GCP UI anyways (i.e. not as code)